rfd/00058-poc/poc.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Let's please not use bash for this. This should be a proper Go program, with unit tests, etc. For example, take a look at https://github.com/gravitational/teleport-plugins/pull/457 where @tcsc's working on the terraform provider publishing.

Reply:
Got an initial build of a Go tool that does this commited/pushed. Still need to clean up the code, add logging, and add tests. Is [this](https://github.com/gravitational/teleport/tree/rfd/0058-package-distribution/build.assets/tooling/cmd/build-apt-repos) along the lines of what you're looking for?